### PR TITLE
Create it.po

### PR DESCRIPTION
**Pull request by albanobattistella**

### CODE REVIEW

File: po/it.po [hunk 1]
```diff
@@ -0,0 +1,183 @@
...
+
+#: src/ui/gtk4/preferences.ui:180
+msgid "End Time"
+msgstr "Ora di fine"
+
+#: src/ui/gtk4/preferences.ui:187
+msgid ""
+"<small>Specifies the time when Bedtime Mode will be automatically turned off."
+"</small>"
+msgstr ""
```
Comment (ionutbortis) [inline po/it.po:67]: Hi and thanks for the translation!

Can you please also add the missing texts? 

Here

File: po/it.po [hunk 2]
```diff
@@ -0,0 +1,183 @@
...
+
+#: src/ui/gtk4/preferences.ui:254
+msgid "On-Demand"
+msgstr ""
```
Comment (ionutbortis) [inline po/it.po:71]: And here.

**Review by ionutbortis — CHANGES_REQUESTED** [review]
It seems that there are two texts that are not translated. Please add also those so I won't mess it up by using Google Translate :sweat_smile: 

Thanks!

File: po/it.po (hunk 2 above)
Comment (ionutbortis) [reply po/it.po]: The "On-Demand" text is still in English. Do you think "Su richiesta" is a valid translation to Italian for this?

Comment (albanobattistella) [reply po/it.po]: Hi,

for me is better "On-demand" but if you prefer if you prefer "su richiesta"..you can do it.
Let me know.
Regards,
Albano

**Pull request merged**